pages/interface/components/Content/index.js

```diff
@@ -27,6 +27,16 @@ import {
 import { KebabHorizontalIcon, LinkIcon, PencilIcon, TrashIcon } from '@/TabNewsUI/icons';
 import { useUser } from 'pages/interface';
 
+const CONTENT_TITLE_PLACEHOLDER_EXAMPLES = [
+  'e.g. A nova era da computação quântica',
```

Comment:
Meu receio com notícias era mencionar uma marca ou produto específico, mas acho que conseguimos simular uma notícia de uma linguagem de programação, como `Nova versão do Python é anunciada com melhorias de desempenho`.